showcase/tests/integration/components/hds/form/file-input/field-test.gts

```diff
@@ -44,23 +49,25 @@ module('Integration | Component | hds/form/file-input/field', function (hooks) {
   });
 
   test('it does not render the yielded contextual components if not provided', async function (assert) {
-    await render(hbs`<Hds::Form::FileInput::Field />`);
+    await render(<template><HdsFormFileInputField /></template>);
     assert.dom('.hds-form-field__label').doesNotExist();
     assert.dom('.hds-form-field__helper-text').doesNotExist();
     assert.dom('.hds-form-field__error').doesNotExist();
   });
 
   test('it automatically provides all the ID relations between the elements', async function (assert) {
     await render(
-      hbs`<Hds::Form::FileInput::Field @extraAriaDescribedBy="extra" as |F|>
+      <template>
+        <HdsFormFileInputField @extraAriaDescribedBy="extra" as |F|>
           <F.Label>This is the label</F.Label>
           <F.HelperText>This is the helper text</F.HelperText>
           <F.Error>This is the error</F.Error>
-        </Hds::Form::FileInput::Field>`,
+        </HdsFormFileInputField>
+      </template>,
     );
     // the control ID is dynamically generated
-    let control = this.element.querySelector('.hds-form-field__control input');
-    let controlId = control.id;
+    const control = find('.hds-form-field__control input');
+    const controlId = control?.id ?? '';
```

Comment:
Ohh, see my previous comment about this pattern. I think this is much more readable, and I would go all the way in and instead of using the `?? ''` I would cast the controlId to a string. What do you think?

Reply:
Just pushed a commit that standardizes across all the tests to use this pattern.